action.yml

```diff
@@ -0,0 +1,153 @@
+name: 'Manifest PR handler'
+description: 'Handles manifest PR related tasks'
+inputs:
+    target-repo:
+      required: false
+      type: string
+      default: nrfconnect/sdk-nrf-testing
+      description: 'target repository for manifest PR'
+    forked-repo:
+      required: false
+      type: string
+      default: nordicbuilder/sdk-nrf-testing
+      description: 'forked repository where changes are pushed'
+    skip-string:
+      required: false
+      type: string
+      default: manifest-pr-skip
+    token:
+      required: true
+      type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: Skip string cancel
+      if: contains(github.event.pull_request.title, inputs.skip-string ) ||
+        contains(github.event.pull_request.body, inputs.skip-string )
+      shell: bash
+      run: |
+        gh run cancel --repo ${{ github.repository }} ${{ github.run_id }} 
```

Comment:
This is suboptimal but unfortunately I didn't find any better way to exit early without giving failure.

Reply:
exit 1?